src/templates/typescript/classes/classTemplate.ts

```diff
@@ -0,0 +1,35 @@
+import MemberVariable from "../../../model/dataTypes/MemberVariable";
+import Handlebars from "../registerPartials";
+import DataType from "../../../model/dataTypes/DataType";
+import EntityMetadata from "../../../model/dataTypes/EntityMetadata";
+import EntityImports from "../../../model/dataTypes/EntityImports";
+
+export const source = `{{> classImport dataType=this }}
+export default class {{ dataType.normalizedName }}{{# if parentDataType }} extends {{ parentDataType.normalizedName }}{{/ if }} {
+  static readonly baseType: string = "{{ metadata.parentTypeName }}";
+  static readonly namespace: string = "{{ metadata.namespace }}";
+  static readonly typeName: string = "{{ metadata.originalTypeName }}";{{# if memberVariables }}
+  {{# each memberVariables }}
+  {{> complexMember member=this }}
+
+  {{/ each }}
+{{/ if }}
+{{# if dataType.primitive }}
+
+  {{> primitiveParse }}
+  {{ else }}
+
+  {{> complexParse }}
+  {{/ if }}
+}
+`;
+
+export interface ITemplateContext {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Interface name must not be prefixed with "I".](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5691315)